features/keychain/module/keychain.js

```diff
@@ -111,6 +111,16 @@ export class Keychain {
     this.#seedLockStatus = Object.create(null)
   }
 
+  removeManySeeds(seeds = []) {
+    const seedIds = getUniqueSeedIds(seeds)
+    for (const id of seedIds) {
+      delete this.#masters[id]
+      delete this.#seedLockStatus[id]
+    }
+
+    return seedIds
```

Comment:
should we only return the ones that were actually deleted?

Reply:
https://github.com/ExodusMovement/exodus-oss/pull/131/commits/fa27fba9d899645d348d582259a7c763b3521fe1